ansible/configs/ocp4-cluster/default_vars_osp.yml

```diff
@@ -29,7 +29,7 @@ ansible_user: cloud-user
 remote_user: cloud-user
 
 # The domain that you want to add DNS entries to
-osp_cluster_dns_zone: blue.osp.opentlc.com
+cluster_dns_zone: blue.osp.opentlc.com
 
 # The base domain
 ocp4_base_domain: "{{ osp_cluster_dns_zone }}"
```

Comment:
Missed this one. Probably change to {{ cluster_dns_zone }}

Reply:
works for me